test/fixtures/resource-limits.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+module.exports = () => {
+  const array = [];
+  while (true) {
+    for (let i = 0; i < 100; i++) {
+      array.push([array]);
+    }
```

Comment:
```suggestion
    array.push([array]);
```

(The extra loop doesn’t do anything inside `while (true)` :slightly_smiling_face:)